zap/src/main/java/org/parosproxy/paros/core/scanner/AbstractPlugin.java

```diff
@@ -1319,6 +1325,7 @@ private AlertBuilder(AbstractPlugin plugin) {
             setReference(plugin.getReference());
             setCweId(plugin.getCweId());
             setWascId(plugin.getWascId());
+            setTags(plugin.getAlertTags());
```

Comment:
The JavaDoc of `newAlert` should be updated to mention this (same comment for passive).